mbed_lstools/lstools_base.py

```diff
@@ -176,6 +176,7 @@ def __init__(self):
         "9012": "SEEED_TINY_BLE",
         "9900": "NRF51_MICROBIT",
         "FFFF": "K20 BOOTLOADER",
+        "OSHW":"Unknown DAPLink Implementation",
```

Comment:
I guess I'm nitpicking now at terminology :) mbed-ls doesn't list Daplink boards, it lists mbed enabled platforms, just like you said. It doesn't require DAPLink.

If not `UNKNOWN_TARGET`, how about `UNKNOWN_PLATFORM` or `COMMUNITY_PLATFORM`? I kind of like `COMMUNITY_PLATFORM` since the manufacturer id is actually "known". It's a general ID that can be used by anyone. That way we're staying consistent with the `platform` terminology used throughout the mbed-ls code base and the command line results. (https://github.com/ARMmbed/mbed-ls/blob/5587839710d02f91138d065a187f3ab1665ecc9e/mbed_lstools/lstools_base.py#L418)